extensions/vscode/src/multiStepInputs/newDeployment.ts

```diff
@@ -367,6 +369,7 @@ export async function newDeployment(
               iconPath: new ThemeIcon("file"),
               label: config.entrypoint,
               description: `(${contentTypeStrings[config.type]})`,
+              detail: result.projectDir,
```

Comment:
We needed to indicate the location of the entrypoints, so now I'm using the second line to hold the workingDir